### PR TITLE
PARAD-M metadata review

### PR DESCRIPTION
**Pull request by cdobs**
Please see the PARAD-M tab of the CGSN Metadata Review for the changes that were made.

### CODE REVIEW

File: calibration/PARADM/CGINS-PARADM-50201__20190328.csv [hunk 1]
```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+50201,CC_bsipar_par_scaling,1,
```
Comment (reedan88) [inline calibration/PARADM/CGINS-PARADM-50201__20190328.csv]: Looks good.

File: calibration/PARADM/CGINS-PARADM-50240__20170419.csv [hunk 2]
```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+50240,CC_bsipar_par_scaling,1,
```
Comment (reedan88) [inline calibration/PARADM/CGINS-PARADM-50240__20170419.csv]: Looks good.

File: calibration/PARADM/CGINS-PARADM-50244__20160926.csv [hunk 3]
```diff
@@ -1,2 +1,2 @@
 serial,name,value,notes
-50244,CC_bsipar_par_scaling,1,
+50244,CC_bsipar_par_scaling,1.00E-06,
```
Comment (reedan88) [inline calibration/PARADM/CGINS-PARADM-50244__20160926.csv]: Looks good.

File: calibration/PARADM/CGINS-PARADM-50151__20160711.csv [hunk 4]
```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+50151,CC_bsipar_par_scaling,1e-06,vendor service
```
[the remark this answers is not in the document]
Comment (pbrickley) [reply calibration/PARADM/CGINS-PARADM-50151__20160711.csv]: Looks ok.

File: calibration/PARADM/CGINS-PARADM-50156__20151013.csv [hunk 5]
```diff
@@ -1,2 +1,2 @@
 serial,name,value,notes
-50156,CC_bsipar_par_scaling,1e-06,
+50156,CC_bsipar_par_scaling,1,not a vendor service but required to correct scaling factor
```
[the remark this answers is not in the document]
Comment (pbrickley) [reply calibration/PARADM/CGINS-PARADM-50156__20151013.csv]: ok

File: calibration/PARADM/CGINS-PARADM-50157__20150509.csv [hunk 6]
```diff
@@ -1,2 +1,2 @@
 serial,name,value,notes
-50157,CC_bsipar_par_scaling,1e-06,
+50157,CC_bsipar_par_scaling,1,not a vendor service but required to correct scaling factor
```
[the remark this answers is not in the document]
Comment (pbrickley) [reply calibration/PARADM/CGINS-PARADM-50157__20150509.csv]: ok

File: calibration/PARADM/CGINS-PARADM-50167__20161107.csv [hunk 7]
```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+50167,CC_bsipar_par_scaling,1,
```
[the remark this answers is not in the document]
Comment (pbrickley) [reply calibration/PARADM/CGINS-PARADM-50167__20161107.csv]: ok

File: calibration/PARADM/CGINS-PARADM-50185__20140702.csv [hunk 8]
```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+50185,CC_bsipar_par_scaling,1.00E-06,
```
[the remark this answers is not in the document]
Comment (pbrickley) [reply calibration/PARADM/CGINS-PARADM-50185__20140702.csv]: ok

File: calibration/PARADM/CGINS-PARADM-50195__20161011.csv [hunk 9]
```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+50195,CC_bsipar_par_scaling,1.00E-06,not a vendor service but required to correct scaling factor
```
[the remark this answers is not in the document]
Comment (pbrickley) [reply calibration/PARADM/CGINS-PARADM-50195__20161011.csv]: ok. Also a change to the scale factor per veh/deployment. I'm giving these the most attention

File: calibration/PARADM/CGINS-PARADM-50199__20150723.csv [hunk 10]
```diff
@@ -1,2 +1,2 @@
 serial,name,value,notes
-50199,CC_bsipar_par_scaling,1.00E-06,
+50199,CC_bsipar_par_scaling,1,
```
[the remark this answers is not in the document]
Comment (pbrickley) [reply calibration/PARADM/CGINS-PARADM-50199__20150723.csv]: ok

File: calibration/PARADM/CGINS-PARADM-50201__20190328.csv (hunk 1 above)
Comment (pbrickley) [reply calibration/PARADM/CGINS-PARADM-50201__20190328.csv]: Recal, but has not been deployed.

File: calibration/PARADM/CGINS-PARADM-50218__20151028.csv [hunk 11]
```diff
@@ -1,2 +1,2 @@
 serial,name,value,notes
-50218,CC_bsipar_par_scaling,1.00E-06,
+50218,CC_bsipar_par_scaling,1,
```
[the remark this answers is not in the document]
Comment (pbrickley) [reply calibration/PARADM/CGINS-PARADM-50218__20151028.csv]: ok

File: calibration/PARADM/CGINS-PARADM-50236__20160901.csv [hunk 12]
```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+50236,CC_bsipar_par_scaling,1.00E-06,
```
[the remark this answers is not in the document]
Comment (pbrickley) [reply calibration/PARADM/CGINS-PARADM-50236__20160901.csv]: ok

[the remark this answers is not in the document]
Comment (pbrickley) [reply calibration/PARADM/CGINS-PARADM-50236__20160901.csv]: ok

[the remark this answers is not in the document]
Comment (pbrickley) [reply calibration/PARADM/CGINS-PARADM-50236__20160901.csv]: So good, I approved it twice.

File: calibration/PARADM/CGINS-PARADM-50238__20190205.csv [hunk 13]
```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+50238,CC_bsipar_par_scaling,1,not a vendor service but required to correct scaling factor
```
[the remark this answers is not in the document]
Comment (pbrickley) [reply calibration/PARADM/CGINS-PARADM-50238__20190205.csv]: The reference in the metadata sheet is wrong, but the cal files is needed. 
OK

File: calibration/PARADM/CGINS-PARADM-50244__20171028.csv [hunk 14]
```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+50244,CC_bsipar_par_scaling,1,not a vendor service but required to correct scaling factor
```
[the remark this answers is not in the document]
Comment (pbrickley) [reply calibration/PARADM/CGINS-PARADM-50244__20171028.csv]: ok

File: calibration/PARADM/CGINS-PARADM-50238__20190205.csv (hunk 13 above)
[the remark this answers is not in the document]
Comment (cdobs) [reply calibration/PARADM/CGINS-PARADM-50238__20190205.csv]: Comment in metadata tracking spreadsheet updated to "CGINS-PARADM-50238__20190205.csv". Was a copy/past error.

File: calibration/PARADM/CGINS-PARADM-50153__20150904.csv [hunk 15]
```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+50153,CC_bsipar_par_scaling,1,vendor service
```
[the remark this answers is not in the document]
Comment (pbrickley) [reply calibration/PARADM/CGINS-PARADM-50153__20150904.csv]: Not sure if this is needed. Need to check the source.

**Pull request merged**